static/app/components/alertBar.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from '@emotion/styled';
+
+const AlertBar = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: ${p => p.theme.headerBackground};
+  background-color: ${p => p.theme.bannerBackground};
+  padding: 6px 30px;
+  font-size: 14px;
+`;
+
+export default AlertBar;
```

Comment:
This sounds pretty generic. Should this have a story in storybook?

Reply:
@Neo-Zhixing Agreed with Evan, maybe we can make a more specific name?

Reply:
Any recommendations? (naming is hard)
`ActionableAlertBar`? `PageAlertBar`?
Btw this is now added to the storybook.

Reply:
PageAlertBar works for me